### PR TITLE
Updated maintainers

### PR DESCRIPTION
**Pull request by AHeise**

### CODE REVIEW

File: generate-stackbrew-library.sh [hunk 1]
```diff
@@ -33,13 +33,14 @@ source common.sh
 cat <<-EOH
 # this file is generated via https://github.com/apache/flink-docker/blob/$(fileCommit "$self")/$self
 
-Maintainers: Patrick Lucas <me@patricklucas.com> (@patricklucas),
+Maintainers: Robert Metzger <rmetzger@apache.org> (@rmetzger),
+             Chesnay Schepler <chesnay@apache.org> (@chesnay),
```
Comment (rmetzger) [inline generate-stackbrew-library.sh:37]: @zentol ok with you?

**Review by rmetzger — APPROVED** [review]
+1 to merge from my point of view. (let's wait for @zentol's approval as well)

**Comment by zentol** [comment]
Do we need to list specific people, or can we just point to the Flink project as a whole?

**Comment by AHeise** [comment]
Having the project is probably more future proof. Not sure why you need to list maintainers here in the first place...

**Comment by zentol** [comment]
> Not sure why you need to list maintainers here in the first place...

Because the docker folks require it: https://github.com/docker-library/official-images#instruction-format

I've found a case where this field [just contains the project](https://github.com/docker-library/official-images/blob/master/library/express-gateway), so let's do that as well.

**Comment by AHeise** [comment]
> > Not sure why you need to list maintainers here in the first place...
> 
> Because the docker folks require it: https://github.com/docker-library/official-images#instruction-format
> 
> I've found a case where this field [just contains the project](https://github.com/docker-library/official-images/blob/master/library/express-gateway), so let's do that as well.

Could you please add a suggestion?

File: generate-stackbrew-library.sh (hunk 1 above)
Comment (rmetzger) [inline generate-stackbrew-library.sh:37]:
```suggestion
Maintainers: The Apache Flink Project <dev@flink.apache.org> (@ApacheFlink)
```

File: generate-stackbrew-library.sh [hunk 2]
```diff
@@ -33,13 +33,14 @@ source common.sh
...
              Ismaël Mejía <iemejia@gmail.com> (@iemejia)
 GitRepo: https://github.com/apache/flink-docker.git
```
Comment (rmetzger) [inline generate-stackbrew-library.sh:39]:
```suggestion
Maintainers: The Apache Flink Project <dev@flink.apache.org> (@ApacheFlink)
GitRepo: https://github.com/apache/flink-docker.git
```

File: generate-stackbrew-library.sh [hunk 3]
```diff
@@ -33,13 +33,12 @@ source common.sh
...
 
-Maintainers: Patrick Lucas <me@patricklucas.com> (@patricklucas),
-             Ismaël Mejía <iemejia@gmail.com> (@iemejia)
+Maintainers: The Apache Flink Project <dev@flink.apache.org> (@ApacheFlink)
 GitRepo: https://github.com/apache/flink-docker.git
 EOH
 
 
-for dockerfile in $(find . -name "Dockerfile"); do
+for dockerfile in $(find . -name "Dockerfile" | sort -r); do
```
Comment (zentol) [inline generate-stackbrew-library.sh:41]: can you move this into a separate commit?

**Review by zentol — APPROVED** [review]
+1, with a small commit massaging

**Comment by AHeise** [comment]
Extracted that into a separate commit and merging.
